code/__HELPERS/unsorted.dm

```diff
@@ -1343,3 +1343,6 @@ B --><-- A
 		c_dist++
 
 	return L
+
+/proc/subtypesof(typepath)

```

Comment:
One in four coding projects suffer from proc overhead, don't worry, it happens to everyone